src/main/java/webserver/entity/Cookies.java

```diff
@@ -0,0 +1,42 @@
+package webserver.entity;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Cookies {
+    private static final String COOKIE_NAME_VALUE_DELIMITER = "=";
+    private static final String COOKIE_DELIMITER = "; ";
+
+    final Map<String, String> cookies;
+    final String path;
+
+    public Cookies(final Map<String, String> cookies, final String path) {
+        this.cookies = new HashMap<>(cookies);
+        this.path = path;
+    }
+
+    public Cookies(final String cookies) {
+        this.cookies = new HashMap<>();
+        this.path = "/";
+        if (cookies == null) {
+            return;
+        }
+
+        final String[] cookiePairs = cookies.split(COOKIE_DELIMITER);
+        for (String cookiePair : cookiePairs) {
+            final String[] pair = cookiePair.split(COOKIE_NAME_VALUE_DELIMITER);
+            this.cookies.put(pair[0], pair[1]);
+        }
+    }
+
+    public String getCookie(final String key) {
+        return cookies.get(key);
+    }
+
+    public String toCookieString() {
+        return cookies.entrySet().stream()
+                .map(entry -> entry.getKey() + COOKIE_NAME_VALUE_DELIMITER + entry.getValue())
+                .collect(Collectors.joining(COOKIE_DELIMITER)) + "; Path=" + path;
```

Comment:
적당히 변수로 추출해주는 것이 조금 더 가독성이 좋아질 것 같아요. 🙂
```suggestion
        String cookiesToString = cookies.entrySet().stream()
            .map(entry -> entry.getKey() + COOKIE_NAME_VALUE_DELIMITER + entry.getValue())
            .collect(Collectors.joining(COOKIE_DELIMITER));
        
        return cookiesToString + "; Path=" + path;
```

Reply:
피드백 반영했습니다! https://github.com/next-step/jwp-was-kakao/pull/31/commits/830438490a6762e2701a0eb2252962a0ba34aa66